### DISCUSSION TITLE
Control CP language with a field in user profile.

### DISCUSSION THREAD

**Discussion by juliencharette** [statamic/cms#7951] [Q&A] [answered]
Does anyone have a solution to let users control their preferred language from their profile page or the control panel in general?

**Comment by jackmcdade** [comment, marked as answer]
You can do that in the Preferences area of the Control Panel.

<img width="234" alt="CleanShot 2023-04-19 at 09 35 18@2x" src="https://user-images.githubusercontent.com/44739/233091719-4dce3963-d1ea-4683-bcad-141d778a3442.png">


<img width="507" alt="CleanShot 2023-04-19 at 09 35 01@2x" src="https://user-images.githubusercontent.com/44739/233091682-beea26c3-8ed0-4fce-a354-cbb76e819545.png">

**Reply by juliencharette** [reply]
Amazing thank you for the quick reply!
